packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -59,11 +57,11 @@ const PersonalDetails = ({
         onSave(current_step, values);
         onCancel(current_step, goToPreviousStep);
     };
+    const citizen = residence || account_settings?.citizen;
```

Comment:
should we add empty string in case of not coming value from account_settings?
something like 
`const citizen = residence || account_settings?.citizen || '';`